### ISSUE TITLE
Icons are broken

### ISSUE THREAD

**Issue by adilold** [Mermade/shins#8]
The icon font in shins is not loaded correctly 
![image](https://user-images.githubusercontent.com/2160676/28926285-3c6b67f8-785f-11e7-9fec-03878a9e3992.png)

I tried copying the font files into my pub folder but that yielded no results. Any ideas?

Thanks

**Comment by MikeRalphson** [comment]
Thanks for reporting! Hmm, I'm seeing CSS errors in Firefox

> Error in parsing value for 'src'.  Skipped to next declaration.  screen.css:392:7
Error in parsing value for 'src'.  Skipped to next declaration. screen.css:393:7

The slate screen.scss doesn't seem to be being built correctly by node-sass.

Have tried updating the node-sass dependency...

**Comment by MikeRalphson** [comment]
```less
@font-face {
  font-family: 'slate';
  src:font-url('slate.eot?-syv14m');
  src:font-url('slate.eot?#iefix-syv14m') format('embedded-opentype'),
    font-url('slate.woff2?-syv14m') format('woff2'),
    font-url('slate.woff?-syv14m') format('woff'),
    font-url('slate.ttf?-syv14m') format('truetype'),
    font-url('slate.svg?-syv14m#slate') format('svg');
  font-weight: normal;
  font-style: normal;
}
```

is getting compiled to

```css
@font-face {
  font-family: 'slate';
  src: font-url("slate.eot?-syv14m");
  src: font-url("slate.eot?#iefix-syv14m") format("embedded-opentype"), font-url("slate.woff2?-syv14m") format("woff2"), font-url("slate.woff?-syv14m") format("woff"), font-url("slate.ttf?-syv14m") format("truetype"), font-url("slate.svg?-syv14m#slate") format("svg");
  font-weight: normal;
  font-style: normal; }
```

Whereas slate has

```css
@font-face{font-family:'slate';
src:url(../fonts/slate.eot?-syv14m);src:url(../fonts/slate.eot?#iefix-syv14m) format("embedded-opentype"),url(../fonts/slate.woff2?-syv14m) format("woff2"),url(../fonts/slate.woff?-syv14m) format("woff"),url(../fonts/slate.ttf?-syv14m) format("truetype"),url(../fonts/slate.svg?-syv14m#slate) format("svg");font-weight:normal;font-style:normal}
```

**Comment by MikeRalphson** [comment]
Thanks again for reporting this @adilhz - I had seen it a couple of times and never dug into why it was happening.

It turns out `node-sass` (which compiles the sccs files to css) doesn't natively support the `font-url` helper function, but there is an add-on module `node-sass-asset-functions` which provides this.

nb fonts will be loaded directly from the `source/fonts` directory, and `buildstyle` has changed from being a script to being a javascript program.

This is working for me now on github pages (https://mermade.github.io/shins), so I will cut a new release.

**Comment by MikeRalphson** [comment]
Should be fixed in v2.0.1-0 but please let me know if not.

**Comment by adilold** [comment]
Can confirm the icons are working. Thanks for the support!
